package.json

```diff
@@ -63,20 +61,6 @@
       "artifactName": "jitsi-meet-${arch}.${ext}",
       "category": "VideoConference;AudioVideo;Audio;Video;Network",
       "description": "Jitsi Meet Desktop App",
-      "desktop": {
```

Comment:
Why are these no longer necessary?

Reply:
They are no longer supported in the new electron-builder, that's why I had to remove them